<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>服务器IP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lindit_serv_addr_2"/>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>服务器Port</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="linedit_server_port_2"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_bind_2">
       <property name="text">
        <string>连接</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QTextEdit" name="textEdit_2"/>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="btn_send">
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
